cosmic-applet-bluetooth/src/config.rs

```diff
@@ -0,0 +1,3 @@
+pub const APP_ID: &str = "com.system76.CosmicAppletNetwork";
```

Comment:
I guess this app_id is a copy&paste error?

Reply:
Yes :sweat_smile: